.github/workflows/build.yml

```
@@ -291,6 +303,89 @@ jobs:
     - name: SSL tests
       run: ./python Lib/test/ssltests.py
 
+  test_hypothesis:
+    name: "Hypothesis Tests on Ubuntu"
+    runs-on: ubuntu-20.04
```

Recently added to the other jobs, let's add here too:
```suggestion
    runs-on: ubuntu-20.04
    timeout-minutes: 60
```

---

Good call.